src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,18 +3,33 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
+    private static final User[] users = new User[]{
             new User("bob@i.ua", "1234"),
             new User("alice@i.ua", "1234")
     };
 
     /**
      * Find user by email. All users are stored in <code>private static final User[] users</code>
+     *
      * @param email - the input parameter
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (User tempUser : users) {
+            if (tempUser.getEmail().equals(email)) {
+                return tempUser;
+            }
+        }
+        return null;
+    }
+
+    public User checkingPassword(String password) {
+        for (User tempUser : users) {
+            if (tempUser.getPassword().equals(password)) {
+                return tempUser;
+            }
+        }
```

Comment:
Duplicated code. remove this method and use `findByEmail` instead of this method